### DISCUSSION TITLE
OpenDTU API - JSON Object question (V, U and D values)

### DISCUSSION THREAD

**Discussion by eddywee** [tbnobody/OpenDTU#1370] [General]
hey guys,

im trying to digg into the api right now and automate some parts after adding a solix (anker) solarbank into my system.
i try to understand a little better the reponses that i receive of my opendtu right now - to be more precise this thing here:

```json
      "AC": {
        "0": {
          "Power": {
            "v": 65.30000305,
            "u": "W",
            "d": 1
```

i assume that "v" represents the value, u is the measurement-unit but i dont have a clue what "d" is standing for?!

Can someone help me out here real quick?

**Comment by Tiese** [comment]
Hallo eddywee,

"d" ist die Anzahl Nachkommastellen.

Hello eddywee,

"d" is the number of decimal places.

**Reply by eddywee** [reply]
thanks @Tiese !!
